### ISSUE TITLE
Muse Sounds not detected by MuseScore or Muse Hub (Linux)

### ISSUE THREAD

**Issue by Arken-Lunk** [musescore/MuseScore#15340]
**Describe the bug**

I downloaded the Strings Muse Sounds, however after restarting Muse Hub they appear as if installing from zero and MuseScore doesn't detect them on the Playback setup nor the mixer. I searched for the downloaded sound files, and they were in fact installed. I am using version 4.0.0 of MuseScore and 1.0.0.392 of Muse Hub. I could not try with more sounds, as I do not currently have enough space to store them. 

**To Reproduce**

1. Download the strings Muse Sound
2. Close Muse Hub
3. Open Muse Hub and MuseScore and the issue appears

**Expected behaviour**

After restarting MuseHub I expected the already downloaded sounds to be detected, as such with MuseScore. 

**Screenshots**

![image](https://user-images.githubusercontent.com/110853531/208309365-50e58dea-0c6a-4612-a75e-b28318c117a0.png)
The Strings MuseSounds appearing as if to be downloaded still after having already done so.

![image](https://user-images.githubusercontent.com/110853531/208309410-0f2cdba7-03fd-4a91-b7c4-2daba542d3dd.png)
MuseScore not detecting any MuseSounds.

![image](https://user-images.githubusercontent.com/110853531/208309445-92636eba-f5e5-4f8b-86b0-800b27a7e170.png)
The folder with the downloaded sounds.

**Platform information**

OS: Zorin OS 15.3 x86_64

**Comment by LuFlo** [comment]
Did you install MuseScore as a flatpak? Then it does not work for some reason... But you can use the AppImage

**Comment by Arken-Lunk** [comment]
> Did you install MuseScore as a flatpak? Then it does not work for some reason... But you can use the AppImage

Yes, I installed it as flatpak as the AppImage requires glibc 2.29, my OS having 2.28, and I didn't want to be forced to upgrade... Is there no other workaround?

**Comment by CodeSpoof** [comment]
Maybe you can do it by granting extended permissions to the flatpak package. You can give permission to specific paths, but I don't know which ones would be needed to accomplish what you are trying to do. If that's not possible, maybe a special solution for flatpak is needed. I myself can't run the AppImage either, but I'm waiting for the DEB package for version 4

**Comment by VoxelPrismatic** [comment]
I'm having the same issue as well.
The problem arises from the fact that Ubuntu only hosts the Musescore 2.3 deb files, meaning the only way we can get updates is through flatpak.

I enabled system file access via `sudo flatpak override --filesystem=host --filesystem=/srv/muse-hub`, and gave read/write permissions to all users for everything under `/srv/muse-hub`. No luck still

Even symlinking `/srv` and `/srv/muse-hub` to `/var/lib/flatpak/app/org.musescore.MuseScore/x86_64/stable/active` and its subfolders didn't work

I checked the AppImage if I could see where it is looking for muse-hub, and I couldn't find it. I think this issue could be resolved sooner if 'musehubPath' was added to the list of diagnostic paths

**Comment by MarcSabatella** [comment]
I'm confused, why not simply use the AppImage instead of messing with third party builds?

**Comment by VoxelPrismatic** [comment]
Because flatpak is easier to update, don't you need to download a new AppImage to get the new version?

~ Daniel Petrov
________________________________

**Comment by Jojo-Schmitz** [comment]
Whatever: the flatpak version is not supported, not by the MuseScore team, so go and complain to the flatpak folks

**Comment by VoxelPrismatic** [comment]
Oh didn't notice this tiny little button next to the header ![image](https://user-images.githubusercontent.com/45671764/218847504-40665c4b-4416-493e-be47-d20a9b1ec5e2.png)

**Comment by MarcSabatella** [comment]
AppImage is updated by simply running it from the command line with the "update" option.  Even works for nightly builds; I update mine daily.

In any case, the point being, the AppImage is the supported version created by the MuseScore team and it should work.  Errors in third party builds would need to be reported to the people who created those builds.

**Comment by cbjeukendrup** [comment]
Closing as stale; if still actual, should be reported via https://support.musehub.com/hc/en-gb/requests/new
